optuna/visualization/_pareto_front.py

```diff
@@ -73,6 +73,39 @@ def objective(trial):
             fig = optuna.visualization.plot_pareto_front(study)
             fig.show()
 
+    Example:
+
+        The following code snippet shows how to plot a 2-dimensional Pareto front
+        of a 3-dimensional study.
+        This example is scalable, e.g., for plotting a 2- or 3-dimensional Pareto front
+        of a 4-dimensional study and so on.
+        The example also contains the option to crop the displayed scale.
```

Comment:
I'm sorry if I missed something, but I guess the current version of the example does not crop the scale.
```suggestion
```

Reply:
correct (in the first PR it was included but later removed).
i removed this comment line. Thanks!

Reply:
Thank you for your update!